### ISSUE TITLE
KeycloakClient not working with relative urls

### ISSUE THREAD

**Issue by juan-dev-santos** [NikiforovAll/keycloak-authorization-services-dotnet#152]
I have my Keycloak service in a relative url behind a proxy ( e.g. https://localhost/identity/ )

Also i have configured the HttpClient used for KeycloakClient  to use _AuthServerUrl_ as base address:

```
services
    .AddHttpClient(BackendIdentityConstants.KeycloakAdminHttpClient,
        (sp, http) =>
        {
            var keycloakOptions = sp.GetRequiredService<IOptions<KeycloakAdminClientOptions>>();

            http.BaseAddress = new Uri(keycloakOptions.Value.AuthServerUrl);
        }
    )
```

But when I try to use the _KeycloakClient.cs_ for creating a user the request uri converts to _https://localhost/admin/myrealm/users_ instead of _https://localhost/identity/admin/myrealm/users_.

This is because the _ApiUrls.AdminApiBase_ is _"/admin"_ instead of _"admin"_, this initial '/' overrides any relative url as it is said in the [BaseAddress documentation](https://learn.microsoft.com/en-us/dotnet/api/system.net.http.httpclient.baseaddress?view=net-8.0#remarks):

> Remarks
>
>When sending a [HttpRequestMessage](https://learn.microsoft.com/en-us/dotnet/api/system.net.http.httprequestmessage?view=net-8.0) with a relative Uri, the message Uri will be added to the [BaseAddress](https://learn.microsoft.com/en-us/dotnet/api/system.net.http.httpclient.baseaddress?view=net-8.0) property to create an absolute Uri.
>
>Note that all characters after the right-most "/" in the base URI are excluded when combined with the message URI. See [RFC 3986](https://tools.ietf.org/html/rfc3986) Uniform Resource Identifier (URI) Generic Syntax specification.

For the time being I have fixed this by removing this initial '/' in _ApiUrls.AdminApiBase_ in a copy of the _ApiUrls_ class.

Maybe this could be fixed by removing that initial '/'?

**Comment by NikiforovAll** [comment]
Yeah, maybe we should consider removing the initial '/', unfortunately, I don't have time to fix it right. Would you like to contribute?

**Comment by juan-dev-santos** [comment]
> Yeah, maybe we should consider removing the initial '/', unfortunately, I don't have time to fix it right. Would you like to contribute?

Sure! It will be a pleasure

Also I have been extending the IKeycloakUserClient and IKeycloakRealmCustomClient adding more api calls, I don't know if that could be also interesting to propose in a PR

**Comment by NikiforovAll** [comment]
Sure, I would be happy to see a contribution from you.
